packages/botfuel-dialog/src/nlus/botfuel-nlu.js

```diff
@@ -144,6 +151,22 @@ class BotfuelNlu extends Nlu {
     const entities = await this.extractor.compute(sentence);
     return entities;
   }
+
+  /**
+   * Spellchecks a sentence.
+   * @param sentence - a sentence
+   * @returns the spellchecked sentence
+   */
+  async spellcheck(sentence) {
+    if (!this.config.nlu || !this.config.nlu.spellchecking) {
+      return sentence;
+    }
+    const key = this.config.nlu.spellchecking;
+    logger.debug('spellcheck', sentence, key);
+    const result = await this.spellchecking.compute({ sentence, key });
+    logger.debug('spellcheck: result', result);
+    return result.correctSentence;
```

Comment:
Here we don't handle spellchecking errors anymore.

I would have use a try/catch block like it was in the Bot class

```javascript
try {	
  const result = await this.spellchecking.compute({ sentence, key });	
  logger.debug('spellcheck: result', result);	
  return result.correctSentence;	
} catch (error) {	
  logger.error('spellcheck: error');	
  if (error.statusCode === 403) {	
    throw new AuthenticationError();	
  }	
  throw error;	
}
```

Or maybe like it's done in the customer-laposte and customer-fdj by returning the sentence parameter in case of error

```javascript
try {	
  ...
} catch (error) {	
  logger.error('spellcheck: error');	
  if (error.statusCode === 403) {	
    throw new AuthenticationError();	
  }	
  return sentence;
}
```